apps/src/templates/lessonOverview/LessonStandards.jsx

```diff
@@ -0,0 +1,143 @@
+import PropTypes from 'prop-types';
+import React, {PureComponent} from 'react';
+import _ from 'lodash';
+import {standardShape} from './lessonPlanShapes';
+
+export default class LessonStandards extends PureComponent {
+  render() {
+    const {standards} = this.props;
+    const standardsByFramework = _(standards)
+      .orderBy('frameworkName')
+      .groupBy('frameworkName')
+      .value();
+    return (
+      <div>
+        {Object.keys(standardsByFramework).map(frameworkName => {
+          const standards = standardsByFramework[frameworkName];
+          return (
+            <Framework
+              name={frameworkName}
+              key={frameworkName}
+              standards={standards}
+            />
+          );
+        })}
+      </div>
+    );
+  }
+}
+LessonStandards.propTypes = {
+  standards: PropTypes.arrayOf(standardShape).isRequired
+};
+
+class Framework extends PureComponent {
+  render() {
+    const {name, standards} = this.props;
+    // Whether all standards in this framework have parent categories.
+    const hasParentCategories = !!standards[0].parentCategoryShortcode;
+    const CategoryClass = hasParentCategories ? ParentCategory : Category;
+    const categoryKey = hasParentCategories
+      ? 'parentCategoryShortcode'
+      : 'categoryShortcode';
+    const standardsByCategory = _(standards)
+      .orderBy(categoryKey, 'shortcode')
+      .groupBy(categoryKey)
+      .value();
+    return (
+      <details key={name}>
+        <summary>{name}</summary>
+        <ul style={{listStyleType: 'none'}}>
+          {Object.keys(standardsByCategory).map(categoryShortcode => {
+            const standards = standardsByCategory[categoryShortcode];
+            return (
+              <CategoryClass
+                key={categoryShortcode}
+                shortcode={categoryShortcode}
+                standards={standards}
+              />
+            );
+          })}
+        </ul>
+      </details>
+    );
+  }
+}
+Framework.propTypes = {
+  name: PropTypes.string.isRequired,
+  standards: PropTypes.arrayOf(standardShape).isRequired
+};
+
+class ParentCategory extends PureComponent {
+  render() {
+    const {shortcode, standards} = this.props;
+    const description = standards[0].parentCategoryDescription;
+    const standardsByCategory = _(standards)
+      .orderBy('categoryShortcode', 'shortcode')
+      .groupBy('categoryShortcode')
+      .value();
+    return (
+      <li key={shortcode}>
+        <details>
+          <summary>
+            {shortcode}
+            {' - '}
+            {description}
+          </summary>
+          <ul style={{listStyleType: 'none'}}>
+            {Object.keys(standardsByCategory).map(categoryShortcode => {
+              const standards = standardsByCategory[categoryShortcode];
+              return (
+                <Category
```

Comment:
Is it ever possible for a category to have multiple parents?

Reply:
good question. not currently. some frameworks have multiple hierarchies along which they categorize their standards, but at present we only plan to provide one hierarchy for each framework. This info on NGSS standards may give an idea for some potential future needs in this space, though: https://www.nextgenscience.org/understanding-standards/understanding-standards